packages/TelegramClientTests-UI.package/TCTUChatMessageListTests.class/instance/testAddingOldMessagesDoesNotScroll.st

```diff
@@ -0,0 +1,14 @@
+testing
+testAddingOldMessagesDoesNotScroll
+
+	| offset loadedMessage secondTCUMessage |
+
+	self fillWithMessages.
+	secondTCUMessage := self subject scroller submorphs second.
+	self subject scrollToShow: secondTCUMessage.
+	
+	loadedMessage := TCTMMocks mockMessage message: 'Loaded Message'.
+	self subject addTop: loadedMessage.
+	offset := self subject vScrollBar value.
+	self subject scrollToShow: secondTCUMessage.
```

Comment:
instead of doing this, I would use a local variable to store the correct offset. This seems to work but the intent, to test wether the offset stays the same, is a bit unclear.